src/components/Header.js

```diff
@@ -1,10 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+
+	const [count, setCount] = useState(0);
+	const userImages=[props.user1,props.user2];
+	const userNames=["고은","정쿨"]
+
+
+
+	return <Wrapper >
+	
+		<ProfileCard onClick={() => setCount(count+1)} >
+			{console.log(count)}
+			<ProfileImage src={userImages[count%2]} width="70" height="70" />
+			<ProfileInfo>
+				<Name>{userNames[count%2]}</Name>
```

Comment:
```suggestion
		<ProfileCard onClick={() => setCount(!count)} >
			{console.log(count)}
			<ProfileImage src={userImages[count]} width="70" height="70" />
			<ProfileInfo>
				<Name>{userNames[count]}</Name>
```
`userImages[count%2]`  이방식도 가능하지만,
count에 숫자가 아닌 true, false를 이용하면 더 보기 좋고 간단할 것 같아요 !
count에 true와 false를 넣었다고 가정하고 수정해보았어요 :)
이미지의 src에서 {userImages[count]} 를 사용하는 것은 삼항연산자 등등으로 바꿔주시면 될 것 같아요.
true가 1, false가 0이니, userImages[count]는 안바꾸어줘도 그대로 작동할지도 모르겠네요.